config/clusters/leap/support.values.yaml

```diff
@@ -11,6 +11,14 @@ grafana:
         hosts:
           - grafana.leap.2i2c.cloud
 prometheus:
+  ingress:
```

Comment:
I think you need the following piece at the top (to be congruent with other clusters) of the file:
```
prometheusIngressAuthSecret:
  enabled: true
```

In addition and related to this specific section, I think this snippet goes under the `server` key, right?

Reply:
Absolutely! Thanks for catching this @damianavila!

Reply:
Done in https://github.com/2i2c-org/infrastructure/pull/1310/commits/d40080b9364a5904ad3057eaa0a37c0f05e86299

Reply:
Nice catch @damianavila! I think d40080b does the trick based on inspection of https://github.com/prometheus-community/helm-charts/blob/b73814645a57e2b13673b2f67dfbcdfd799ee946/charts/prometheus/templates/server/ingress.yaml#L2.

This would have been caught by promethues helm chart if they provided a strict schema!

Reply:
> This would have been caught by promethues helm chart if they provided a strict schema!

Should we open a follow-up issue for that, @consideRatio?